app/views/nsm/steps/work_item_delete/edit.html.erb

```diff
@@ -1,35 +1,49 @@
 <% title t('.page_title') %>
-<% decision_step_header %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <%= govuk_error_summary(@form_object) %>
     <h1 class="govuk-heading-l"><%= t('.heading') %></h1>
 
     <table class="govuk-table">
-      <caption class="govuk-table__caption govuk-table__caption--m"><%=  @form_object.record.completed_on&.to_fs(:long) || t('.unknown_date') %></caption>
+      <caption class="govuk-table__caption govuk-table__caption--m"></caption>
       <thead class="govuk-table__head">
         <tr class="govuk-table__row">
-          <th scope="col" class="govuk-table__header govuk-!-width-one-half"><%= t('.headers.work_type') %></th>
-          <th scope="col" class="govuk-table__header govuk-!-width-one-quarter"><%= t('.headers.period') %></th>
+          <th scope="col" class="govuk-table__header govuk-!-width-one-quarter"><%= t('.headers.cost_type') %></th>
+          <th scope="col" class="govuk-table__header govuk-!-width-one-quarter"><%= t('.headers.date') %></th>
           <th scope="col" class="govuk-table__header govuk-!-width-one-quarter"><%= t('.headers.fee_earner') %></th>
+          <th scope="col" class="govuk-table__header govuk-!-width-one-quarter"><%= t('.headers.time_spent') %></th>
         </tr>
       </thead>
       <tbody class="govuk-table__body">
         <tr class="govuk-table__row">
           <td class="govuk-table__cell"><%= check_missing(@form_object.record.work_type) { t("laa_crime_forms_common.nsm.work_type.#{@form_object.record.work_type}") } %></td>
-          <td class="govuk-table__cell"><%= check_missing(format_period(@form_object.record.time_spent))  %></td>
+          <td class="govuk-table__cell"><%= @form_object.record.completed_on&.to_fs(:short_stamp) || t('.unknown_date') %></td>
```

Comment:
Rather than `t('.unknown_date')` I think you want to wrap this in a `check_missing`, that way if it's missing it gets a pink `Incomplete` tag like the other fields